### ISSUE TITLE
Uncaught TypeError: Cannot read property 'hasTooltips' of undefined

### ISSUE THREAD

**Issue by clojj** [atom-haskell/ide-haskell#134]
[Enter steps to reproduce below:]

1. open .hs source (simple module with main)
2. open haskell-ide-repl
3. press esc-key in the repl-input field, see screenshot:
![screen shot 2015-12-08 at 22 35 40](https://cloud.githubusercontent.com/assets/4367963/11669281/3106e354-9dfc-11e5-8db6-ee23c63c6d98.png)

**Atom Version**: 1.2.4
**System**: Mac OS X 10.11.2
**Thrown From**: [ide-haskell](https://github.com/atom-haskell/ide-haskell) package, v1.3.6

### Stack Trace

Uncaught TypeError: Cannot read property 'hasTooltips' of undefined

```
At /Users/jwin/.atom/packages/ide-haskell/lib/ide-haskell.coffee:157

TypeError: Cannot read property 'hasTooltips' of undefined
    at atom-text-editor.disposables.add.atom.commands.add.ide-haskell:close-tooltip (/Users/jwin/.atom/packages/ide-haskell/lib/ide-haskell.coffee:157:58)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:533:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:356:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeydown (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:99:36)
    at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:3:61)
```

### Commands

```
  2x -0:36.2.0 ide-haskell:close-tooltip (atom-text-editor.editor.is-focused)
     -0:36.2.0 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -0:36.2.0 core:cancel (atom-text-editor.editor.is-focused)
     -0:31.7.0 ide-haskell-repl:toggle (atom-text-editor.editor.is-focused)
     -0:26.3.0 command-palette:toggle (atom-text-editor.editor.is-focused)
 12x -0:22.3.0 core:move-up (atom-text-editor.editor.mini.is-focused)
     -0:17.8.0 editor:consolidate-selections (atom-text-editor.editor.mini.is-focused)
     -0:17.8.0 core:cancel (atom-text-editor.editor.mini.is-focused)
     -0:10.1.0 snippets:next-tab-stop (atom-text-editor.editor.ide-haskell-repl.is-focused)
     -0:10.1.0 snippets:expand (atom-text-editor.editor.ide-haskell-repl.is-focused)
     -0:10.1.0 editor:indent (atom-text-editor.editor.ide-haskell-repl.is-focused)
     -0:09.2.0 core:backspace (atom-text-editor.editor.ide-haskell-repl.is-focused)
     -0:05.6.0 editor:newline (atom-text-editor.editor.ide-haskell-repl.is-focused)
     -0:04.0 core:backspace (atom-text-editor.editor.ide-haskell-repl.is-focused)
     -0:03.3.0 editor:newline (atom-text-editor.editor.ide-haskell-repl.is-focused)
     -0:01.3.0 ide-haskell:close-tooltip (atom-text-editor.editor.ide-haskell-repl.is-focused)
```

### Config

```json
{
  "core": {
    "audioBeep": false,
    "disabledPackages": [
      "conceal",
      "spell-check",
      "cursor-history",
      "cabal",
      "wrap-guide"
    ],
    "packagesWithKeymapsDisabled": [
      "ide-haskell",
      "term3",
      "cursor-history"
    ],
    "themes": [
      "one-dark-ui",
      "necro-syntax"
    ]
  },
  "ide-haskell": {
    "stylishHaskellPath": "/Users/jwin/Library/Haskell/bin/stylish-haskell",
    "cabalPath": "/usr/local/bin/cabal",
    "onCursorMove": "Hide Tooltip"
  }
}
```

### Installed Packages

```coffee
# User
asciidoc-preview, v0.6.0
atom-wallaby, v1.0.7
autocomplete-haskell, v0.6.2
compare-files, v0.6.2
haskell-ghc-mod, v1.2.0
haskell-pointfree, v0.1.2
highlight-selected, v0.11.1
hindent, v0.0.1
ide-haskell, v1.3.6
ide-haskell-cabal, v1.0.0
ide-haskell-repl, v0.0.7
language-haskell, v1.5.1
necro-syntax, v1.0.0
project-manager, v2.6.5
term3, v0.19.2

# Dev
No dev packages
```

**Comment by lierdakil** [comment]
Thanks for catching this. Fixed in 1.3.7

**Comment by clojj** [comment]
Fastest bugfix.. ever :)
